cfg/Tomb1Main_gameflow.json5

```diff
@@ -19,6 +19,17 @@
     // [0.45, 1.0, 1.0] is default TombATI filter.
     "water_color": [0.45, 1.0, 1.0],
 
+    // The distance (in tiles) at which objects and the world
+    // start to fade into blackness.
+    // The default hardcoded value in TR1 is 12.
+    // The default hardcoded value in TombATI is 80.
```

Comment:
In TombATI fade is 72, not 80, and the whole feature is off by default to "not ruin the ambience" or something like that.

Reply:
Fixed